### ISSUE TITLE
<fdp-multi-combobox> component doesn't work properly

### ISSUE THREAD

**Issue by OrangeLi514** [SAP/fundamental-ngx#8473]
#### Is this a bug, enhancement, or feature request?
this is a bug

#### Briefly describe your proposal.
There are two issues with this <fdp-multi-combobox> component:
1. The elements in the mutual-combox can not be deleted. (see attachment) **Can't delete element "Budget Report User".**
2. User input text is still presenting in the combox even after auto-complete option has been selected. (see attachment) **The "y" is still there**

<img width="959" alt="Screen Shot 2022-07-28 at 6 08 36 PM" src="https://user-images.githubusercontent.com/110207830/181662223-5aeeaf8b-2151-4da5-acc1-c765b3dbffd0.png">

https://user-images.githubusercontent.com/110207830/181853031-e312d629-c670-452d-bbd2-31ebfc80d58a.mp4

#### Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)

versions of Angular:ng13
versions of Fundamental Library: v0.35.3
#### If this is a bug, please provide steps for reproducing it.
issue1:
choose select options and then delete it
issue2:
type any character in the combobox, then select one options from auto-complete list. Then click on any places, you can see the character is still presenting in combobox.

**Comment by droshev** [comment]
@OrangeLi514 the second was a feature requested [here](https://github.com/SAP/fundamental-ngx/issues/6718)
